packages/dd-trace/test/ci-visibility/exporters/agentless/exporter.spec.js

```diff
@@ -136,7 +134,7 @@ describe('CI Visibility Agentless Exporter', () => {
     })
     it('will not allow skippable request if ITR configuration fails', (done) => {
       // request will fail
-      delete process.env.DD_APP_KEY
+      delete process.env.DD_API_KEY
```

Comment:
I guess this was an oversight before that API key wasn't deleted?

Reply:
well I was testing the failure of the ITR settings request, which failed as long as any app or api key were missing. Removing app key then was enough